src/quemb/molbe/chemfrag.py

```diff
@@ -58,21 +59,12 @@
 
 # We would like to have a subtype of Sequence that also behaves generically
 # so that we could write
-# `SeqOverFrag[AOIdx]` for a sequence that contains all AO indices in a fragment
-# or `SeqOverAtom[AOIdx]` for a sequence that contains all AO indices in an atom,
-# where the Sequence types are different, i.e. a function that takes a `SeqOverFrag`
-# would neither accept a `SeqOverAtom` nor a generic `Sequence`.
+# `Sequence[AOIdx]` for a sequence that contains all AO indices in a fragment
+# or `Sequence[AOIdx]` for a sequence that contains all AO indices in an atom,
+# where the Sequence types are different, i.e. a function that takes a `Sequence`
+# would neither accept a `Sequence` nor a generic `Sequence`.
```

Comment:
When you changed from `SeqOverFrag` and `SeqOverAtom` to `Sequence`, were the changes also addressed in the content of this comment block? I am a bit confused if I follow this comment clearly...

Reply:
Good catch. I fixed the comment.